archiva-modules/plugins/maven2-repository/src/main/java/org/apache/archiva/dependency/tree/maven2/Maven3DependencyTreeBuilder.java

```diff
@@ -280,12 +280,8 @@ private RepositorySystemSession newRepositorySystemSession( RepositorySystem sys
         DependencySelector depFilter = new AndDependencySelector( new ExclusionDependencySelector() );
         session.setDependencySelector( depFilter );
 
-        LocalRepository localRepo = new LocalRepository( localRepoDir );
         session.setLocalRepositoryManager(
-            new SimpleLocalRepositoryManager( localRepoDir ) );// system.newLocalRepositoryManager( localRepo ) );
-
-        //session.setTransferListener(  );
-        //session.setRepositoryListener( n );
+            new SimpleLocalRepositoryManager( localRepoDir ) );
```

Comment:
Other option is to remove `String msg` completely. Or should it be logged as INFO?